### PR TITLE
Use get-host-info.sub.js in no-cross-origin-autofocus.sub.html

### PR DESCRIPTION
**Pull request by alijuma**
WebKit's test infrastructure doesn't support domains like www1.localhost.
This change uses get_host_info().HTTP_REMOTE_ORIGIN to choose an
appropriate cross-origin domain that works with both WPT infrastructure
as well as with WebKit's test infrastructure.

### CODE REVIEW

**Comment by zcorpan** [comment]
I think it is unfortunate that WebKit's test infrastructure is such that WPT's intended host names don't work. Is there a plan for the longer term to make it work?

For now, I think PRs like this one should be accepted so that WebKit can use the tests. But new tests will continue to be written that won't work for WebKit, unless either there is a policy change for WPT or WebKit's test infrastructure is changed so that WPT host names work.

cc @web-platform-tests/admins for thoughts on this.

**Comment by Hexcles** [comment]
> For now, I think PRs like this one should be accepted so that WebKit can use the tests. 

I agree. BTW, I think it's more appropriate to cc @web-platform-tests/wpt-core-team .

File: html/semantics/forms/autofocus/no-cross-origin-autofocus.sub.html [hunk 1]
```diff
@@ -36,7 +37,7 @@ <h1>Autofocus shouldn't work in cross-origin iframe.</h1>
         }
       }));
       document.getElementById("child").src =
-          "http://{{domains[www]}}:{{ports[http][0]}}/html/semantics/forms/autofocus/resources/child-autofocus.html";
+          get_host_info().HTTP_REMOTE_ORIGIN + "/html/semantics/forms/autofocus/resources/child-autofocus.html";
```
Comment (Ms2ger) [inline html/semantics/forms/autofocus/no-cross-origin-autofocus.sub.html]: We can remove `.sub` from the file name now, I think.

Comment (alijuma) [reply html/semantics/forms/autofocus/no-cross-origin-autofocus.sub.html]: Thanks, done.

**Comment by youennf** [comment]
LGTM.
Another way to fix this, which is less intrusive, is to use http://{{hosts[alt][]}} instead of http://{{domains[www]}}.
In WebKit will be translated to http://127.0.0.1:8800 by the WPT server.
There is an action (on me or WebKit team) to update wherever possible domains[www] to hosts[alt].

**Review by youennf — APPROVED** [review]

**Pull request merged**